context.c

```diff
@@ -100,10 +101,17 @@ static ssize_t iio_snprintf_context_xml(char *ptr, ssize_t len,
 					const struct iio_context *ctx)
 {
 	ssize_t ret, alen = 0;
-	unsigned int i;
+	unsigned int i, major, minor;
+	char git_tag[64];
+
+	ret = (ssize_t) iio_context_get_version(ctx, &major, &minor, git_tag);
```

Comment:
curious: is the cast needed for other platforms (OSx or Windows)?

Reply:
Casts must be explicit when casting to a smaller type, a different qualifier, or a different signness. I guess in this case it could be implicit.

Reply:
Yes, in this case the cast is implicit but I do not now if some compilers in other platforms would complain about this. That was why I asked

Reply:
I removed this, and Windows didn't complain, so it's fine.